app/models/info_request/batch_pagination.rb

```diff
@@ -0,0 +1,22 @@
+# Methods to handle pagination within an associated InfoRequestBatch
+module InfoRequest::BatchPagination
+  def next_in_batch
+    return nil unless info_request_batch
+    info_request_batch.info_requests[index_in_batch + 1] || first_in_batch
```

Comment:
Are we certain the info_requests are always order the same way?

Reply:
Pretty sure, yeah. Could explicitly order by ID here?